src/stateMachine/InterfacingStateMachine.cpp

```diff
@@ -3,10 +3,9 @@
 #include "stateMachineLib/StateMachine.h"
 
 
-InterfacingStateMachine::InterfacingStateMachine(BYTE maxStates, BYTE initialState) : 
-	StateMachine(maxStates, initialState), interfaceImpl()
+InterfacingStateMachine::InterfacingStateMachine(Interface* anInterface, BYTE maxStates, BYTE initialState) :
+	StateMachine(maxStates, initialState), interface(anInterface)
```

Comment:
Will this perform a copy?

Reply:
It will copy the pointer, but not where the pointer points to.